### ISSUE TITLE
no unique ID in 3-node cluster 

### ISSUE THREAD

**Issue by jchristianj** [dougiteixeira/proxmoxve#203]
### Checklist

- [X] I have updated to the latest available Proxmox VE Custom Integration version, [see the latest version here](https://github.com/dougiteixeira/proxmoxve/releases/latest).

### Describe the issue you are experiencing

Hi, since I have 3 proxmox nodes in a cluster and using the integration, there is an issue during start-up of home assistant:

Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:612
Integration: Sensor (documentation, issues)
First occurred: 13:04:34 (4 occurrences)
Last logged: 13:04:34

Platform proxmoxve does not generate unique IDs. ID 6f9b0639ea5cff4e24688eaf61d1e092_/dev/nvme0n1_node already exists - ignoring sensor.disk_proxmox2_dev_nvme0n1_node
Platform proxmoxve does not generate unique IDs. ID 6f9b0639ea5cff4e24688eaf61d1e092_/dev/nvme0n1_size already exists - ignoring sensor.disk_proxmox2_dev_nvme0n1_size
Platform proxmoxve does not generate unique IDs. ID 6f9b0639ea5cff4e24688eaf61d1e092_/dev/nvme0n1_temperature already exists - ignoring sensor.disk_proxmox2_dev_nvme0n1_temperature
Platform proxmoxve does not generate unique IDs. ID 6f9b0639ea5cff4e24688eaf61d1e092_/dev/nvme0n1_power_cycles already exists - ignoring sensor.disk_proxmox2_dev_nvme0n1_power_cycles

### In which version of Home Assistant Core do you have the problem?

core-2023.12.3

### What version of Proxmox VE Custom Integration has the issue?

3.2.2.

### What version of Proxmox VE do you have the problem?

8.1.3.

### Diagnostics information

[home-assistant_proxmoxve_2023-12-15T12-15-34.450Z.log](https://github.com/dougiteixeira/proxmoxve/files/13685032/home-assistant_proxmoxve_2023-12-15T12-15-34.450Z.log)

**Comment by alexruffell** [comment]
I have the same errors, and I too have 3 nodes in a cluster:

```
2023-12-30 09:21:11.926 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_health already exists - ignoring binary_sensor.disk_maximus_dev_nvme0n1_health
2023-12-30 09:21:11.926 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_health already exists - ignoring binary_sensor.disk_maximus_dev_nvme1n1_health
2023-12-30 09:21:11.926 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_health already exists - ignoring binary_sensor.disk_maximus_dev_nvme0n1_health
2023-12-30 09:21:11.926 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_health already exists - ignoring binary_sensor.disk_maximus_dev_nvme1n1_health
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_node already exists - ignoring sensor.disk_maximus_dev_nvme0n1_node
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_size already exists - ignoring sensor.disk_maximus_dev_nvme0n1_size
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_temperature already exists - ignoring sensor.disk_maximus_dev_nvme0n1_temperature
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_power_cycles already exists - ignoring sensor.disk_maximus_dev_nvme0n1_power_cycles
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_node already exists - ignoring sensor.disk_maximus_dev_nvme1n1_node
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_size already exists - ignoring sensor.disk_maximus_dev_nvme1n1_size
2023-12-30 09:21:11.938 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_temperature already exists - ignoring sensor.disk_maximus_dev_nvme1n1_temperature
2023-12-30 09:21:11.939 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_power_cycles already exists - ignoring sensor.disk_maximus_dev_nvme1n1_power_cycles
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_node already exists - ignoring sensor.disk_maximus_dev_nvme0n1_node
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_size already exists - ignoring sensor.disk_maximus_dev_nvme0n1_size
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_temperature already exists - ignoring sensor.disk_maximus_dev_nvme0n1_temperature
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme0n1_power_cycles already exists - ignoring sensor.disk_maximus_dev_nvme0n1_power_cycles
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_node already exists - ignoring sensor.disk_maximus_dev_nvme1n1_node
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_size already exists - ignoring sensor.disk_maximus_dev_nvme1n1_size
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_temperature already exists - ignoring sensor.disk_maximus_dev_nvme1n1_temperature
2023-12-30 09:21:11.940 ERROR (MainThread) [homeassistant.components.sensor] Platform proxmoxve does not generate unique IDs. ID 90287b040ca0b945180705f1a00ea8c1_/dev/nvme1n1_power_cycles already exists - ignoring sensor.disk_maximus_dev_nvme1n1_power_cycles
```

**Comment by drewancil** [comment]
I have this as well.  Happy to provide info but the above likely covers it

**Comment by dougiteixeira** [comment]
I've included the fix for this in #227. It will be available in the next version.
